react/components/AccordionFilterGroup.js

```diff
@@ -1,11 +1,16 @@
 import React from 'react'
 import { useIntl } from 'react-intl'
+import classNames from 'classnames'
```

Comment:
Is this import being used?

Reply:
Yes, it is being used on line 40. 
You can check it by seeing the whole file